### DISCUSSION TITLE
[Security] Can private vulnerability reporting be enabled?

### DISCUSSION THREAD

**Discussion by rikaardhosein** [tprouvot/Salesforce-Inspector-reloaded#657] [General]
The process is described here: https://docs.github.com/en/code-security/security-advisories/working-with-repository-security-advisories/configuring-private-vulnerability-reporting-for-a-repository

If not, can another method for reporting vulnerabilities be added to the README? Thank you!

**Comment by tprouvot** [comment]
Hi @rikaardhosein,
The option has been enabled !
